app/assets/stylesheets/home.css.scss

```diff
@@ -5,48 +5,51 @@
 $splash-image-height: 600px;
 
 .splash-block {
-    @include full-width-block($off-white);
-    background-color: transparent;
+  @include full-width-block($off-white);
+  background-color: transparent;
 }
 
 .splash-images {
-    z-index: -20;
-    margin-top: $topbar-height + 1;
-    position: absolute;
-    width: 100%;
-    top: 0;
-    .splash-image {
-        background-size: cover;
-        height: $splash-image-height;
-    }
+  z-index: -20;
+  margin-top: $topbar-height + 1;
+  position: absolute;
+  width: 100%;
+  top: 0;
+	
+  .splash-image {
+    background-size: cover;
+    height: $splash-image-height;
+  }
 }
 
 .splash-overlay {
-    z-index: -10;
-    margin-top: $topbar-height + 1;
-    position: absolute;
-    width: 100%;
-    height: $splash-image-height;
-    top: 0;
-    background-color: rgba(0, 0, 0, 0.2);
-
+  z-index: -10;
+  margin-top: $topbar-height + 1;
+  position: absolute;
+  width: 100%;
+  height: $splash-image-height;
+  top: 0;
+  background-color: rgba(0, 0, 0, 0.25);
 }
 
 .splash-title {
-    height: $splash-image-height;
-    background-color: transparent;
-    h1 {
-        padding-top: 17%;
-        color: white;
-        font-weight: 800;
-        font-size: 5.0em;
-    }
-    h2 {
-        color: white;
-        font-weight: 600;
-        font-size: 1.5em;
-    }
-    .button-row {
-        padding-top: 50px;
-    }
+  height: $splash-image-height;
+  background-color: transparent;
+
+  h1 {
+    padding-top: 17%;
+    color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.